packages/amplify-util-uibuilder/src/commands/utils/syncAmplifyUiBuilderComponents.ts

```diff
@@ -1,3 +1,4 @@
+// es-lint-disable-file spellcheck/spell-checker
```

Comment:
We are actively working on improving our linter. Please do not include this and instead add words into the dictionary here: https://github.com/aws-amplify/amplify-cli/blob/dev/.eslint-dictionary.json